components/caseStudyDetails/FigmaDesign.tsx

```diff
@@ -11,7 +11,7 @@ const FigmaDesign = ({ project }: { project: string }) => {
         High-Fidelity Figma Design
       </div>
       <Image
-        className="h-full max-h-[47.9375rem] w-full max-w-[90rem] 2xl:mx-auto"
+        className="h-full w-full"
```

Comment:
This was a good lesson we learned during office hours! Thanks for bringing this up and asking questions!